client/my-sites/checklist/wpcom-checklist/index.jsx

```diff
@@ -111,7 +112,8 @@ export default connect( ( state, ownProps ) => {
 	const isEligibleForChecklist = isEligibleForDotcomChecklist( state, siteId );
 	const isSection = isSiteSection( state );
 	const taskStatuses = get( getSiteChecklist( state, siteId ), [ 'tasks' ] );
-	const taskList = getTaskList( taskStatuses, designType );
+	const isSiteUnlaunched = isUnlaunchedSite( state, siteId );
+	const taskList = getTaskList( taskStatuses, designType, isSiteUnlaunched );
```

Comment:
Seems like this was accidentally removed in https://github.com/Automattic/wp-calypso/pull/27926/